Stratis.Bitcoin/Features/Consensus/PosConsensusValidator.cs

```diff
@@ -10,6 +10,23 @@
 
 namespace Stratis.Bitcoin.Features.Consensus
 {
+    /// <summary>
```

Comment:
I am happy to document this here for now.
though my plan in the future is to break all the rules to dedicated classes.

Reply:
Still the documentation should be in one place. I would like to avoid having to go to 20 files to be able to find out all rules that apply.

Reply:
yes thats true I would even pull it out to its own readme.